app/src/main/java/com/ivoryworks/cipunchingbag/MainActivity.java

```diff
@@ -1,13 +1,36 @@
 package com.ivoryworks.cipunchingbag;
 
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.widget.TextView;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        Integer i = null;
+        i.toString();
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ivoryworks/CI-PunchingBag/blob/dfea560f798d8c0bdbe728ea240a1d9c31a1d7c8/app/src/main/java/com/ivoryworks/cipunchingbag/MainActivity.java#L17"></span>Null pointer dereference of i in com.ivoryworks.cipunchingbag.MainActivity.onCreate(Bundle)</td>
    </tr>
  </tbody>
</table>